src/utils/hooks/useRouter.tsx

```diff
@@ -6,7 +6,7 @@ import { BasicStringObject } from '../types'
 interface RouterResult {
   goBack: () => void
   pathname: string
-  push: (path: string) => void
+  push: (path: string, state?: any) => void
```

Comment:
Hmmm where is this used?

Reply:
Passing from the Data view menu to dataview-table, and also passing from Data-table to Data-view Detail (though that's in the next PR)

Reply:
I have an explanation for each instance of it in comments.